pages/1.8/administration/installing/ent/custom/troubleshooting/index.md

```diff
@@ -14,76 +14,76 @@ enterprise: false
 # <a name="general"></a>General troubleshooting approach
 
 1.  Verify you have a valid IP detect﻿⁠⁠⁠⁠ script, functioning DNS resolvers to bind the DC/OS services to, and that all nodes are synchronized with NTP.
-    
+
     ## <a name="ip-detect-script"></a>IP detect script
-    
+
     You must have a valid [ip-detect](/1.8/administration/installing/ent/custom/advanced/) script. You can manually run `ip-detect` on all the nodes in your cluster or check `/opt/mesosphere/bin/detect_ip` on an existing installation to ensure that it returns a valid IP address. A valid IP address does not have:
-    
+
     - extra lines
     - white space
-    - special or hidden characters 
-    
+    - special or hidden characters
+
     It is recommended that you use the `ip-detect` [examples](/1.8/administration/installing/ent/custom/advanced/).
-      
+
     ## DNS resolvers
-      
+
     You must have working DNS resolvers, specified in your [config.yaml](/1.8/administration/installing/ent/custom/configuration-parameters/#resolvers) file. It is recommended that you have forward and reverse lookups for FQDNs, short hostnames, and IP addresses. It is possible for DC/OS to function in environments without valid DNS support, but the following _must_ work to support DC/OS services, including Spark:
-    
+
     - `hostname -f` returns the FQDN
     - `hostname -s` returns the short hostname
-    
+
     You should sanity check the output of `hostnamectl` on all of your nodes as well.
 
-    When troubleshooting problems with a DC/OS installation, you should explore the components in this sequence: 
-    
-    1. Exhibitor 
-    1. Mesos master 
-    1. Mesos DNS 
-    1. DNS Dispatcher 
+    When troubleshooting problems with a DC/OS installation, you should explore the components in this sequence:
+
+    1. Exhibitor
+    1. Mesos master
+    1. Mesos DNS
+    1. DNS Dispatcher (Spartan)
     1. DC/OS Marathon
-    1. Jobs 
+    1. Jobs
     1. Admin Router
 
     Be sure to check that all services are up and healthy on the masters before checking the agents.
-    
+
     ### NTP
-    
+
     Network Time Protocol (NTP) must be enabled on all nodes for clock synchronization. By default, during DC/OS startup you will receive an error if this is not enabled. You can check if NTP is enabled by running one of these commands, depending on your OS and configuration:
-    
+
     ```bash
     ntptime
     adjtimex -p
     timedatectl
     ```
 
 1. Ensure that firewalls and any other connection-filtering mechanisms are not interfering with cluster component communications. TCP, UDP, and ICMP must be permitted.
-    
-   Ensure that services that bind to port `53`, which is required by DNS Dispatcher (`dcos-spartan.service`), are disabled and stopped. For example: 
-   
+
+   Ensure that services that bind to port `53`, which is required by DNS Dispatcher (`docs-spartan.service`), are disabled and stopped. For example:
```

Comment:
minor typo ... it should be `dcos` not `docs`

Reply:
Fixed!